server/runtime/src/main/java/org/infinispan/server/configuration/ServerConfigurationParser.java

```diff
@@ -1299,9 +1303,22 @@ private void parseEndpoints(XMLExtendedStreamReader reader, ConfigurationBuilder
       while (reader.hasNext() && (reader.nextTag() != XMLStreamConstants.END_ELEMENT)) {
          reader.handleAny(holder);
       }
+      if (endpoint.connectors().isEmpty()) {
+         endpoint.addConnector(HotRodServerConfigurationBuilder.class).name("hotrod-" + socketBinding);
+         RestServerConfigurationBuilder rest = endpoint.addConnector(RestServerConfigurationBuilder.class).name("rest-" + socketBinding);
```

Comment:
We should abstract the creation of the default Hot Rod and REST endpoint names to a static method in the `HotRodServerConfigurationParser` and `RestServerConfigurationParser`.

Reply:
Agreed.